bin/pulumilocal

```diff
@@ -23,17 +24,62 @@ if os.path.isdir(os.path.join(PARENT_FOLDER, ".venv")):
 
 # define global constants
 TRUE_STRINGS = ["1", "true"]
+CONFIG_STRATEGIES = ("overwrite", "override", "separation")
+DEFAULT_CONFIG_STRATEGY = "overwrite"
+LOCAL_STACK_NAME = os.environ.get("LOCAL_STACK_NAME") or "localstack"
+CONFIG_STRATEGY = os.environ.get("CONFIG_STRATEGY") if os.environ.get("CONFIG_STRATEGY") in CONFIG_STRATEGIES else DEFAULT_CONFIG_STRATEGY
```

Comment:
Currently we're doing the following:
1. if strategy is missing or invalid we display the chosen strategy,
2. a message that no valid configuration strategy has been chosen,
3. in the above message we're stating that we're falling back to the default strategy (overwrite) which is equivalent with the old behaviour
4. and setting the fallback strategy
5. continue the run
Exiting with a warning would break any existing CIs imo.

Reply:
I am glad we have the logging in place 👍 but regarding 

> Exiting with a warning would break any existing CIs imo.

this is a new flag and we are defaulting to the existing strategy if not set, so I don't think this is the case.

Reply:
As we've agreed the fall back behaviour is only temporary until the next release.